bluesky/suspenders.py

```diff
@@ -525,3 +525,123 @@ def _get_justification(self):
+        suspend_instrument_in_use = SuspendWhenChanged(instrument_in_use)
+        RE.install_suspender(suspend_instrument_in_use)
+    
+    Parameters
+    ----------
+
+    signal : `ophyd.Signal`
+        The signal to watch for changes to determine if the
+        scan should be suspended
+
+    expected_value : number (float or int)
+        RunEngine operations will be suspended when signal deviates
+        from this value.  If `None` (default), set to value of 
+        ``signal`` when object is created.
+
+    allow_resume : bool
+        Should RunEngine be allowed to resume once ``signal.value == expected``
+        again?  Default value of ``False`` is expected for intended use case.
+    
+    sleep : float, optional
+        How long to wait in seconds after the resume condition is met
+        before marking the event as done.  Defaults to 0
+
+    pre_plan : iterable or iterator or generator function, optional
+            a generator, list, or similar containing `Msg` objects
+
+    post_plan : iterable or iterator or generator function, optional
+            a generator, list, or similar containing `Msg` objects
+
+    tripped_message : str, optional
+        Message to include in the trip notification
+
+    Example EPICS database for 2-BM-A and 2-BM-B::
+
+        record(mbbo, "2bm:instrument_in_use") {
+            # instrument team sets this
+            # For additional field names, see
+            # https://epics.anl.gov/EpicsDocumentation/AppDevManuals/RecordRef/Recordref-25.html#HEADING25-15
+            field(DESC, "instrument using beam now")
+            field(ZRST, "none")
+            field(ONST, "2-BM-A")
+            field(TWST, "2-BM-B")
+            # THST
+            # FRST
+            # FVST
+            # ...
+        }
+    
+    NOTE: Always make the zero choice (``ZRST``) in the mbbo record to be 'none'.
+    This allows the instrument staff to designate that *no* instrument is allowed 
+    to control the shared hardware.
+    """
+    
+    def __init__(self, signal, *, 
+                expected_value=None,
+                allow_resume=False,
+                sleep=0, pre_plan=None, post_plan=None, tripped_message='',
+                **kwargs):
+        
+        self.expected_value = expected_value or signal.value
+        self.allow_resume = allow_resume
+        super().__init__(signal, 
+            sleep=sleep, 
+            pre_plan=pre_plan, 
+            post_plan=post_plan, 
+            tripped_message=tripped_message,
+            **kwargs)
+
+    def _should_suspend(self, value):
+        return value != self.expected_value
+
+    def _should_resume(self, value):
+        return self.allow_resume and value == self.expected_value
+
+    def _get_justification(self):
+        if not self.tripped:
+            return ''
+
+        just = f'Signal {self._sig.name}'
+        just += f', got "{self._sig.get()}"'
+        just += f', expected "{self.expected_value}"'
```

Comment:
This `just` variable can be simplified as such:
```py
just = (f'Signal {self._sig.name}'
        f', got "{self._sig.get()}"'
        f', expected "{self.expected_value}"')

```

Reply:
My innate style will change.  Slowly.  Thanks.

Reply:
No worries, just wanted to be more consistent with the rest of multiline strings in the codebase :).